packages/data-transport-layer/src/services/l1-ingestion/service.ts

```diff
@@ -114,8 +110,11 @@ export class L1IngestionService extends BaseService<L1IngestionServiceOptions> {
     this.l1IngestionMetrics = registerMetrics(this.metrics)
 
     if (typeof this.options.l1RpcProvider === 'string') {
-      this.state.l1RpcProvider = FallbackProvider(this.options.l1RpcProvider, {
-        'User-Agent': 'data-transport-layer',
+      this.state.l1RpcProvider = new StaticJsonRpcProvider({
```

Comment:
`proxyd` should be able to handle the retry logic now, so making this change should be fine